### ISSUE TITLE
Adding webp images to manifest

### ISSUE THREAD

**Issue by paulodeon** [iampava/imagemin-webp-webpack-plugin#18]
I'm using Rails with Webpacker - while the output of imageminwebp appears in the emitted files the webp files dont appear in the manifest.

In rails you need them in the manifest so their full urls i.e. with hash can be generated

I've tried many different approaches to get them in there, is there something I'm missing?

**Comment by iampava** [comment]
Unfortunately I have zero experience with Rails or Webpacker so I don't really know what goes wrong. I guess it doesn't "play well" with Webpacker but it's just an assumption.

However, can you maybe modify this manifest yourself, after the build process? Maybe you could create a custom script which runs after build and modifies the contents 🤔

**Comment by agrberg** [comment]
@paulodeon Not sure if you ever found a solution to this but I've encountered it myself and found this post which I added to in the webpacker repository. https://github.com/rails/webpacker/issues/1940#issuecomment-605704939

**Comment by agrberg** [comment]
I updated the linked issue to Webpacker ☝️as I was able to get this working!

tl;dr [This gist](https://gist.github.com/agrberg/53f7ccc61f35e3af28f53d19eef59675) contains my current `environment.js` file which adds the emitted files to the manifest by using `WebpackAssetsManifest` [hooks feature](https://github.com/webdeveric/webpack-assets-manifest#hooks).
